### PR TITLE
add openbsd build support

### PR DESCRIPTION
**Pull request by kremlin-**
minor

### CODE REVIEW

**Comment by rssnyder** [comment]
@kremlin- thanks for the PR.

Can you add openbsd to the github workflow as well to create a release?

```
goos: [linux, windows, darwin]  ->  goos: [linux, windows, darwin, openbsd]
```

On this line: https://github.com/rssnyder/discord-stock-ticker/blob/master/.github/workflows/deploy.yml#L34

**Comment by kremlin-** [comment]
👍

File: Makefile [hunk 1]
```diff
@@ -3,9 +3,12 @@ build-linux:
 
 build-osx:
 	env GOOS=darwin GOARCH=amd64 go build -o ./bin/ticker
+	
+build-openbsd
```
[the remark this answers is not in the document]
Comment (kremlin-) [reply Makefile:7]: 🤦

**Review by rssnyder — APPROVED** [review]
LG2M, thanks!

**Pull request merged**